environment.yaml

```diff
@@ -3,6 +3,7 @@
 name: nebari-demo
 channels:
 - conda-forge
+- pyviz
```

Comment:
```suggestion
```
Atm conda-store needs to modified to accept other channels besides `conda-forge`